liquibase-core/src/main/java/liquibase/parser/core/formattedsql/FormattedSqlChangeLogParser.java

```diff
@@ -350,7 +350,8 @@ public DatabaseChangeLog parse(String physicalChangeLogLocation, ChangeLogParame
                         changeLogParameters.expandExpressions(StringUtil.stripEnclosingQuotes(authorGroup), changeLog);
 
                     changeSet =
-                       new ChangeSet(changeSetId, changeSetAuthor, runAlways, runOnChange, logicalFilePath, context, dbms, runWith, runInTransaction, changeLog.getObjectQuotingStrategy(), changeLog);
+                       new ChangeSet(changeSetId, changeSetAuthor, runAlways, runOnChange, DatabaseChangeLog.normalizePath(logicalFilePath), context, dbms, runWith, runInTransaction, changeLog.getObjectQuotingStrategy(), changeLog);
+                    changeSet.setStoredFilePath(logicalFilePath);
```

Comment:
This is kind of a misuse of the storedFilePath field. That field is really intended to represent the file path as specified in the databasechangelog table. I don't think it hurts anything by specifying it here, but I don't think it's totally necessary if it would be better to remove it.

Reply:
I removed this as it broke rollback.